sql/core/src/test/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionNodeSuite.scala

```diff
@@ -100,6 +117,8 @@ class SqlScriptingExecutionNodeSuite extends SparkFunSuite with SharedSparkSessi
       case loopStmt: LoopStatementExec => loopStmt.label.get
       case leaveStmt: LeaveStatementExec => leaveStmt.label
       case iterateStmt: IterateStatementExec => iterateStmt.label
+      case forStmt: ForStatementExec => forStmt.label.get
+      case _: SingleStatementExec => "SingleStatementExec"
```

Comment:
maybe do a `statement: SingleStatementExec if statement.logicalPlan.isInstanceOf[DropVariable] => "DropVariable"` just to be more strict with the tests?